### PR TITLE
[openweathermap] Semantics added to channel types

### PR DESCRIPTION
**Pull request by lolodomo**
Also fix few categories to use only official categories

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/thing/channel-types.xml [hunk 1]
```diff
@@ -399,6 +440,10 @@
 		<label>Forecasted Temperature</label>
 		<description>Forecasted outdoor temperature.</description>
 		<category>Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/thing/channel-types.xml:444]: I am wondering here: A forecast is not really a measurement. I'd therefore rather refrain from adding this tag.
We might want to think of something like a "forecast" tag for the tag library instead. Wdyt?

Comment (lolodomo) [reply bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/thing/channel-types.xml:444]: You're right, a forecast is not exactly a measurement.
I will add a new Point "Forecast" in the semantics tag.

Comment (lolodomo) [reply bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/thing/channel-types.xml:444]: I am going to replace "Measurement" by "Status" for forecast channels.

**Comment by kaikreuzer** [comment]
I'd suggest we park this PR until we finish the discussion on https://github.com/openhab/openhab-core/issues/1791. Putting "Status" on everything where we don't have dedicated semantic tags for does not feel right to me and I wouldn't want to see this to be done for 300+ bindings now...

**Comment by lolodomo** [comment]
@kaikreuzer : to make progress, I propose to only keep tags that are obvious (in particular measurements) and suppress "Status" where we don't have dedicated semantic tags. Can we at least do that ?

**Comment by kaikreuzer** [comment]
Sorry for dropping the ball here. 😩 

>  to make progress, I propose to only keep tags that are obvious (in particular measurements like temperature, humidity, pressure, ...) and suppress "Status" where we don't have dedicated semantic tags. Can we at least do that ?

Yes, let's do that, please!

**Comment by lolodomo** [comment]
@kaikreuzer : PR is updated.

**Comment by lolodomo** [comment]
As there was a long pause in the discussion, I do not remember if this is appropriate to add measurement) temperatures on min temperature or dew point temperature? 
Some of the bindings have already done.

**Review by kaikreuzer — APPROVED** [review]

**Pull request merged**